sites/all/modules/paragraphs/reol_news_paragraph/reol_news_paragraph.module

```diff
@@ -22,29 +22,143 @@ function reol_news_paragraph_field_formatter_info() {
 }
 
 /**
+ * Implements hook_menu().
+ */
+function reol_news_paragraph_menu() {
+  $items = array();
+
+  $items['reol_news_paragraph/ajax/%/%/%/%/%'] = array(
```

Comment:
A comment or two explaining what these arguments are about would be nice

Reply:
They're documented on the function, which grabs them all? I don't think it's common to document arguments in hook_menu().